imports/message-read-receipt/client/readReceipts.html

```diff
@@ -6,7 +6,11 @@
 		<ul class="read-receipts">
 			{{#each receipts}}
 				<li class="read-receipts__user background-transparent-dark-hover">
-					{{> avatar username=user.username}}
+                                        {{#if user}}
```

Comment:
@renatobecker  Do you want an avatar with guest name or username, cause if it is the guest name, it will change accordingly, otherwise it will always be **G**

Reply:
We don't have Avatars for Livechat users.
It's ok showing the default avatar as you just described.